app/controllers/application_controller.rb

```diff
@@ -52,7 +55,8 @@ def routing_error
   end
 
   def log_error(exception)
-    Raven.capture_exception(exception) if ENV['SENTRY_DSN'].present?
+    # report the original 'cause' of the exception when present
+    Raven.capture_exception(exception.cause.presence || exception) if ENV['SENTRY_DSN'].present?
```

Comment:
Yes, if you look at client/base.rb I'm raising a generic exception for any of the more granular Faraday::ClientError exceptions, which could be any one of 5 exceptions.